File: src/main/resources/static/app/users.controller.js
(function () {
    'use strict';

    angular
        .module('app')
        .controller('UsersController', UsersController);

    UsersController.$inject = ['$http'];

    function UsersController($http) {
        var vm = this;

        vm.users = [];
        vm.getAll = getAll;
        vm.getLogin = getLogin;
        vm.deleteU = deleteU;
        vm.getAllActive = getAllActive;
        vm.getAllNActive = getAllNActive;

        init();

        function init(){
            getAllActive();
        }

        function getAll(){
            var url = "/Users/all";
            var usersPromise = $http.get(url);
            usersPromise.then(function(response){
                vm.users = response.data;
            });
        }
        function getAllActive(){
            var url = "/Users/active";
            var usersPromise = $http.get(url);
            usersPromise.then(function(response){
                vm.users = response.data;
            });
        }
        function getAllNActive(){
            var url = "/Users/notactive";
            var usersPromise = $http.get(url);
            usersPromise.then(function(response){
                vm.users = response.data;
            });
        }

        function getLogin(login){
            var url = "/Users/find/" + login;
            var usersPromise = $http.get(url);
            usersPromise.then(function(response){
                vm.users = response.data;
            });
        }

        function deleteU(id){
            var url = "/Users/delete/" + id;
            $http.post(url).then(function(response){
                vm.users = response.data;
            });
        }
    }
})();

File: src/main/java/com/feli/mycrm/UsersController.java
package com.feli.mycrm;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.*;

import java.util.List;

@RestController
@RequestMapping(value = "/Users")
public class UsersController {

    private UsersRepository usersRepository;

    @Autowired
    public UsersController(UsersRepository usersRepository){
        this.usersRepository = usersRepository;
    }

    @RequestMapping(value = "/active", method = RequestMethod.GET)
    public List<Users> getAllActive(){
        return usersRepository.findAllActive();
    }
    @RequestMapping(value = "/notactive", method = RequestMethod.GET)
    public List<Users> getAllNActive(){
        return usersRepository.findAllNActive();
    }
    @RequestMapping(value = "/all", method = RequestMethod.GET)
    public List<Users> getAll(){
        return usersRepository.findAll();
    }

    @RequestMapping(value = "/find/{login}", method = RequestMethod.GET)
    public List<Users> getLogin(@PathVariable String login){
        return usersRepository.findByLoginContaining(login);
    }

    @RequestMapping(value = "/create", method = RequestMethod.POST)
    public List<Users> create(@RequestBody Users users){
        usersRepository.save(users);

        return usersRepository.findAll();
    }

    @RequestMapping(value = "/delete/{id}", method = RequestMethod.POST)
    public List<Users> remove(@PathVariable long id){
        usersRepository.deleteById(id);
        //usersRepository.setIsDeleted(id);
        return usersRepository.findAllActive();
    }
}
